Preserve the two local fixes that the 3.3.x branch used to carry on top of
the embedded nrgljubljana subtree (upstream commit 59d30dc, version 2020.09):

  - Remove use of std::unary_function (removed in C++17)
    Original commit: 09bd91ad "Remove use of std::unary_function"

  - Stop overriding modern CMake's FindBLAS/FindLAPACK with the bundled
    2020-vintage copies in CMake/BLAS_LAPACK/
    Original commit: 72a1afbf "Port to latest triqs unstable" (nrgljubljana/ part)

diff --git a/c++/boost/numeric/conversion/detail/converter.hpp b/c++/boost/numeric/conversion/detail/converter.hpp
index c84269e7..a51f0388 100644
--- a/c++/boost/numeric/conversion/detail/converter.hpp
+++ b/c++/boost/numeric/conversion/detail/converter.hpp
@@ -450,9 +450,7 @@ namespace boost { namespace numeric { namespace convdetail
   // Trivial Converter : used when (cv-unqualified) T == (cv-unqualified)  S
   //
   template<class Traits>
-  struct trivial_converter_impl : public std::unary_function<  BOOST_DEDUCED_TYPENAME Traits::argument_type
-                                                              ,BOOST_DEDUCED_TYPENAME Traits::result_type
-                                                            >
+  struct trivial_converter_impl : public std::function<BOOST_DEDUCED_TYPENAME Traits::result_type(BOOST_DEDUCED_TYPENAME Traits::argument_type)>
                                  ,public dummy_range_checker<Traits>
   {
     typedef typename Traits::result_type   result_type   ;
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 6b53a192..b574e411 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -183,7 +183,6 @@ if(Mathematica_FOUND)
   message(STATUS "Mathematica executable ${Mathematica_KERNEL_EXECUTABLE}")
 endif()

-list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/CMake/BLAS_LAPACK)
 # BLAS/Lapack
 find_package(LAPACK REQUIRED) # also looks up BLAS
 add_library(blas_lapack INTERFACE)
